samples/bluetooth/ble_hids_keyboard/sample.yaml

```diff
@@ -19,3 +19,23 @@ tests:
       - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice/mcuboot
       - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice/mcuboot
     tags: ci_build
+  sample.ble_hids_keyboard_ra_protection:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice
+      - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice
+      - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice
+      - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice/mcuboot
+      - bm_nrf54l15dk/nrf54l10/cpuapp/s115_softdevice/mcuboot
+      - bm_nrf54l15dk/nrf54l15/cpuapp/s115_softdevice/mcuboot
+    platform_allow:
+      - bm_nrf54l15dk/nrf54l05/cpuapp/s115_softdevice
```

Comment:
Do we need to build this for all platforms?

Reply:
That depends. We don't have a consistent policy on how to go about this.

Reply:
Yes, we need to decide. Building this configuration six times per sample seems a bit of a waste as nothing in the board itself should affect it, other than the memory size.